internal/sys/syscall.go

```diff
@@ -19,7 +19,7 @@ var ENOTSUPP = syscall.Errno(524)
 func BPF(cmd Cmd, attr unsafe.Pointer, size uintptr) (uintptr, error) {
 	// Prevent the Go profiler from repeatedly interrupting the verifier,
 	// which could otherwise lead to a livelock due to receiving EAGAIN.
-	if cmd == BPF_PROG_LOAD {
+	if cmd == BPF_PROG_LOAD || cmd == BPF_PROG_RUN {
```

Comment:
@ti-mo should we move PROG_LOAD retry into the caller as well, to make PROG_LOAD and PROG_RUN consistent?

Reply:
I think that was my initial proposal, but having this behaviour here is better from a clarity/doc perspective, since there's no way to misuse it this way.